src/EnergyPlus/SolarShading.cc

```diff
@@ -12603,7 +12618,7 @@ void CalcInteriorWinTransDifSolInitialDistribution(EnergyPlusData &state,
 
                 // Accumulate transmitted diffuse solar for reporting
                 state.dataHeatBalSurf->SurfWinInitialDifSolInTrans(HeatTransSurfNum) +=
-                    (DifSolarTransW / state.dataSurface->Surface(HeatTransSurfNum).Area);
+                    (SolarTrans_ViewFactor - DifSolarReflW / state.dataSurface->Surface(HeatTransSurfNum).Area);
```

Comment:
Are the parentheses wrong on this one?  It is different from the others.

Reply:
@Myoldmopar You are right, the parentheses are wrong. It seem that it should be:
(SolarTrans_ViewFactor - DifSolarReflW) / state.dataSurface->Surface(HeatTransSurfNum).Area;
I did not run example to confirm this. However, there are other parts using the same equation:
https://github.com/NREL/EnergyPlus/blob/f3c55b306880c1e210aeaafc77f97b212ce44643/src/EnergyPlus/SolarShading.cc#L12441-L12442
and
https://github.com/NREL/EnergyPlus/blob/f3c55b306880c1e210aeaafc77f97b212ce44643/src/EnergyPlus/SolarShading.cc#L12511-L12512